.github/workflows/trigger-a-build-elsewhere.yml

```diff
@@ -0,0 +1,32 @@
+name: Trigger Janus build
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Trigger workflow
+      uses: actions/github-script@v6
+      with:
+        github-token: ${{ secrets.PAT_ACTIONS_JANUS }}
```

Comment:
Worth commenting this and perhaps linking to docs on how to regenerate when it expires?

Reply:
documented this in the script